cdci_data_analysis/analysis/instrument.py

```diff
@@ -486,6 +489,14 @@ def get_product_query_name(self, product_type):
         else:
             return self.query_dictionary[product_type]
 
+
+    def check_instrument_access(self, roles, email):
+        if self.restricted_access:
```

Comment:
this kind of two-level access specification is not quite what I meant. Rather "development" can be an attribute of InstrumetQuery and all logic done in `instrumet_query.check_instrument_roles` - it's easier to see access logic then. Any disadvantage of this approach?

Reply:
But my idea is to already have the access login within `instrument_query.check_instrument_roles`, as I am doing. But I had never seen the `development`/ `restricted_access` as an attribute of the InstrumentQuery, rather of the Instrument itself. 

I will try what you propose

Reply:
You mean you never seen it where? We are just adding it here, no?

Reply:
Simply I meant that I was never considering the option of having it as an attribute of `InstrumetQuery`

And yes, I added it only here